spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/PartTreeSpannerQuery.java

```diff
@@ -57,26 +58,30 @@ protected List executeRawResult(Object[] parameters) {
 		}
 		if (this.tree.isDelete()) {
 			return this.spannerTemplate
-					.performReadWriteTransaction(transactionTemplate -> {
-						List<T> entitiesToDelete = SpannerStatementQueryExecutor
-								.executeQuery(this.entityType, this.tree, parameters,
-										transactionTemplate, this.spannerMappingContext);
-						transactionTemplate.deleteAll(entitiesToDelete);
-
-						List result = null;
-						if (this.queryMethod.isCollectionQuery()) {
-							result = entitiesToDelete;
-						}
-						else if (this.queryMethod.getReturnedObjectType() != void.class) {
-							result = Collections.singletonList(entitiesToDelete.size());
-						}
-						return result;
-					});
+					.performReadWriteTransaction(getDeleteFunction(parameters));
 		}
 		return SpannerStatementQueryExecutor.executeQuery(this.entityType, this.tree,
 						parameters, this.spannerTemplate, this.spannerMappingContext);
 	}
 
+	private Function<SpannerTemplate, List> getDeleteFunction(Object[] parameters) {
```

Comment:
Honestly I think having a separate private method that returns a `Function` I think is messier than just having the contents of this method as the anonymous function directly.

Reply:
@ChengyuanZhao 
Feel free to create a PR, I was just fixing checkstyle issues.

Reply:
But was this specific change flagged by checkstyle?

Reply:
@ChengyuanZhao 
it was number of returns per method issue